webserver/website/src/components/installButton.vue

```diff
@@ -0,0 +1,28 @@
+<template>
+    <button v-if="showInstallButton" @click="install" class="btn btn-primary">App installieren</button>
+</template>
+
+<script>
+export default {
+  name: 'installButton',
+  data: function () {
+    return {
+      showInstallButton: false,
+      deferredPrompt: null
+    }
+  },
+  mounted: function () {
+    window.addEventListener('beforeinstallprompt', (e) => {
```

Comment:
Man kann auf Firefox in Mobile des auch Installieren, aber der Button wird nicht angezeigt. 😢

Reply:
Mach das doch als Issue auf. Warum Firefox sich nicht wie Chrome verhält, verstehe ich nicht. Aber vermutlich muss man für jeden Browser das nochmal einzeln überarbeiten. Und auf IOS geht das grundsätzlich irgendwie gar nicht.